### DISCUSSION TITLE
Interpolate parameter from csv data

### DISCUSSION THREAD

**Discussion by presvetli** [pybamm-team/PyBaMM#2011] [Q&A] [answered]
For  `Negative electrode OCP [V]` I have some .csv data (x=sto, y=ocp). I want to skip function fitting for that data, if it is possible.
In some parameter function file, can I read csv data in X, Y list and use numpy to find interpolate parameter from table --> numpy.interp(`sto`, X, Y)?
`sto`: :class:`pybamm.Symbol`  Electrode stochiometry

**Comment by ashish17khandelwal** [comment, marked as answer]
please use
pybamm.interpolant(x,y,sto)
